.github/workflows/build-android.yml

```diff
@@ -93,11 +101,15 @@ jobs:
           sed -i 's/defonce version ".*"/defonce version "${{ steps.ref.outputs.version }}"/g' src/main/frontend/version.cljs
           sed -i 's/versionName ".*"/versionName "${{ steps.ref.outputs.version }}"/g' android/app/build.gradle
 
+      - name: Set Build Environment Variables
+        run: |
+          echo "ENABLE_FILE_SYNC=${{ inputs.build-target == 'true' || github.event.inputs.enable-file-sync == 'true' }}" >> $GITHUB_ENV
```

Comment:
Why `inputs.build-target == 'true'` is counted into `ENABLE_FILE_SYNC=true`

Reply:
typo